utils/README.md

```diff
@@ -1,15 +1,21 @@
-# BUFR Utilities
 
-This is a collection of commonly used BUFR utilities:
-[readbp](#readbp) - Read prepbufr file and print reports
-[readmp](#readmp) - Read any BUFR file and dump contents of each subset one at a time
-[binv](#binv) - Print BUFR file inventory by message type
-[sinv](#sinv) - Print inventory of satellite data by platform and instrument type
-[cmpbqm](#cmpbqm) - Print inventory of prepbufr observations by variable, report type and quality marks
+#### Sample Utilities
+@brief Collection of commonly-used utilities based on the library
+
+| Utility | Description   |
+|---------|--------------------------------------------------------------------------|
+| [readbp](#readbp) | Read prepbufr file and print each report one at a time |
+| [readmp](#readmp) | Read BUFR file containing embedded DX tables, and print each report one at a time |
+| [binv](#binv) | Print inventory of BUFR file by message type |
+| [sinv](#sinv) | Print inventory of BUFR satellite data file by platform and instrument type |
+| [cmpbqm](#cmpbqm) | Print inventory of observations from prepbufr file by variable, report type and quality mark |
+
+<br>
 
 ---
 
-## readbp
+<div id="readbp"/>
+### readbp
```

Comment:
Its unable to interpret `###`

Reply:
here and in every heading.

Reply:
Hmm, if I just type `### readbp` at the beginning of a new line right here in this GitHub window, it works fine:

### readbp

but for some reason it doesn't work when rendered on github.com.  Maybe it needs an extra blank line between the `<div>` tag and itself?

Reply:
Yep, that's exactly what it was (grrr!!).   OK, I'll push up yet another commit to fix this, and maybe I'll go take a nap while I'm waiting for mac-os to run again ;-)